scim.sgnl.yaml

```diff
@@ -0,0 +1,450 @@
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  
+  Roles:
+    parent: Users  
+    displayName: SCIMUserRole
+    externalId: roles
+    description: Entity representing SCIM user roles
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+
+  Photo:
+    parent: Users  
+    displayName: SCIMUserPhoto
+    externalId: photos
+    description: Entity representing SCIM user photos
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  
+  Certificates:
+    parent: Users  
+    displayName: SCIMUserCertificate
+    externalId: certificates
+    description: Entity representing SCIM user certificates
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: display
+        externalId: display
+        type: String  
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+
+relationships:
+  # Manager relationship
+  UserManager:
+    name: Manager
+    fromAttribute: Users.$..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
+    toAttribute: Users.id
+  
+  # The internal `parent` relationship cannot be directly referenced in `path` relationship.
+  # Create a dummy relationship to represent the `parent` relationship.
+  # The fromAttribute and toAttribute does not matter. Only entities should be specified correctly.
+  # We rely on the `parent` relationship already created during ingestion.
+  ParentOfGroupMember: 
+    name: parent
+    fromAttribute: groups.displayName
+    toAttribute: Users.id
+  MemberOf:
+    name: MemberOf
+    fromAttribute: groups.value
+    toAttribute: Groups.id
```

Comment:
To add to @rlenglet's point, using `externalId` might result in bad UX. We should eventually transition to human readable names.

In this case, `groups` is the `externalId` of a `GroupMember` entity. `Groups` is the externalId of the `Group` entity - this would be even more ugly and confusing had it been JSONPaths.

cc: @shash-sgnl, @ivanfong